test/e2e/fedora_test.go

```diff
@@ -12,10 +12,11 @@ import (
 	"os"
 	"path/filepath"
 	"runtime"
+	"github.com/ulikunitz/xz"
 )
 
 const (
-	fedoraBaseDirEndpoint = "https://kojipkgs.fedoraproject.org/compose/cloud/latest-Fedora-Cloud-39/compose"
+	fedoraBaseDirEndpoint = "https://kojipkgs.fedoraproject.org/compose/cloud/latest-Fedora-Cloud-40/compose"
```

Comment:
mind changing to 41?

Reply:
Just noticed this as well in my image update Pr and tried to do a quick fix https://github.com/containers/libhvee/pull/197

Anyway it is complicated as there is no f41 on the server: i.e. 404 https://kojipkgs.fedoraproject.org/compose/cloud/latest-Fedora-Cloud-41/

I asked in the Fedora cloud matrix to see if this is a bug or not on their end. So Sticking to f40 seems to be the right call for now

Reply:
https://pagure.io/releng/issue/12521

Reply:
works for me